packages/extension-chakra-storefront/config/default.json

```diff
@@ -11,42 +15,142 @@
         }
     },
     "defaultSite": "<SITE_ID>",
+    "defaultAppLocale": "<DEFAULT_APP_LOCALE>",
+    "defaultSiteTitle": "<DEFAULT_SITE_TITLE>",
     "einsteinAPI": {
         "host": "https://api.cquotient.com",
         "einsteinId": "<EINSTEIN_ID>",
         "siteId": "<EINSTEIN_SITE_ID>",
         "isProduction": false
     },
+    "maxCacheAge": 900,
     "pages": {
-        "Account": "/account",
-        "Cart": "/cart",
-        "Checkout": "/checkout",
-        "CheckoutConfirmation": "/checkout/confirmation/:orderNo",
-        "Home": "/",
-        "Login": "/login",
-        "Registration": "/registration",
-        "ResetPassword": "/reset-password",
-        "LoginRedirect": "/callback",
-        "ProductDetail": "/product/:productId",
-        "ProductList": ["/search", "/category/:categoryId"]
+        "Account": {
+            "path": "/account",
+            "orderSearchParam": {"limit": 10, "offset": 0, "sort": "best-matches", "refine": []}
+        },
+        "Cart": {
+            "path": "/cart"
+        },
+        "Checkout": {
+            "path": "/checkout",
+            "shippingCountryCode": [
+                {"value": "CA", "label": "Canada"},
+                {"value": "US", "label": "United States"}
+            ]
+        },
+        "CheckoutConfirmation": {
+            "path": "/checkout/confirmation/:orderNo"
+        },
+        "Home": {
+            "path": "/",
+            "productLimit": 10,
+            "mainCategory": "newarrivals"
+        },
+        "Login": {
+            "path": "/login"
+        },
+        "Registration": {
+            "path": "/registration"
+        },
+        "ResetPassword": {
+            "path": "/reset-password"
+        },
+        "LoginRedirect": {
+            "path": "/callback"
+        },
+        "ProductDetail": {
+            "path": "/product/:productId"
+        },
+        "ProductList": {
+            "path": ["/search", "/category/:categoryId"],
+            "imageViewType": "large",
+            "selectedAttributedId": "color",
+            "filterAccordionSate": "filters-expanded-index"
+        }
+    },
+    "search": {
+        "defaultLimitValues": [25, 50, 100],
+        "defaultSearchParams": {"limit": 25, "offset": 0, "sort": "best-matches", "refine": []},
+        "recentSearchKey": "recent-search-key",
+        "recentSearchLimit": 5,
+        "recentSearchMinLength": 3
+    },
+    "siteAliases": {
+        "RefArch": "us",
+        "RefArchGlobal": "global"
     },
-    "siteAliases": {},
     "sites": [
         {
-            "id": "<SITE_ID>",
```

Comment:
Please revert the `sites` section to these placeholders because we don't know how the user environments are set up.

Reply:
thanks for catching that. Code is reverted